Description: Fix the appearance of the URL bar dropdown with dark themes
Author: Chris Coulson <chris.coulson@canonical.com>
Forwarded: not-needed

diff --git a/toolkit/themes/gnomestripe/global/autocomplete.css b/toolkit/themes/gnomestripe/global/autocomplete.css
--- a/toolkit/themes/gnomestripe/global/autocomplete.css
+++ b/toolkit/themes/gnomestripe/global/autocomplete.css
@@ -145,20 +145,18 @@ treechildren.autocomplete-treebody::-moz
 .autocomplete-richlistitem[selected="true"] {
   background-color: Highlight;
   color: HighlightText;
 }
 
 .autocomplete-richlistitem {
   padding: 1px 2px;
   color: MenuText;
-}
-
-.autocomplete-richlistitem:not(:first-child) {
-  border-top: 1px solid ThreeDLightShadow;
+  border-top: 3px solid transparent;
+  border-bottom: 3px solid transparent; 
 }
 
 .ac-site-icon {
   width: 16px; 
   height: 16px;
   margin: 0 5px -3px 3px;
 }
 
